### DISCUSSION TITLE
Disable compilation for certain files

### DISCUSSION THREAD

**Discussion by aneziac** [nvarner/typst-lsp#439] [Q&A]
Is there a way to disable compilation for certain files? If I've set the extension compilation on saving, is there some flag I can set so that when I save a file it doesn't get exported to pdf?

My use case is that I have configuration files which I include in my main files. These get regularly updated and upon saving, I always generate a blank pdf. Not a big deal but I'm wondering if this can be avoided.

**Comment by uben0** [comment]
I have a similar need. I use typst on many small text file just to have nice markup syntax, but I don't need a pdf file. Is there a way to disable compilation with, for instance, a comment at the top of the file?
